### PR TITLE
Added tests for float type

### PR DESCRIPTION
**Pull request by Nattalli**
Added test cases with float type numbers

### CODE REVIEW

File: tests/test_main.py [hunk 1]
```diff
@@ -49,7 +54,9 @@ def test_distance_class_str(kilometers, output):
     [
         (50, "Distance(km=50)"),
         (100, "Distance(km=100)"),
-        (300, "Distance(km=300)")
+        (300, "Distance(km=300)"),
+        (12.5, "Distance(km=12.5)"),
+        (.6, "Distance(km=0.6)")
```
Comment (Abnormaltype) [inline tests/test_main.py:59]:
```suggestion
        (0.6, "Distance(km=0.6)")
```

File: tests/test_main.py [hunk 2]
```diff
@@ -363,11 +410,18 @@ def test_distance_class_le_number(kilometers, kilometers2, result):
...
-    assert len(distance) == distance.km, (
-        f"'len()' for instance should return instance.km"
-    )
+    if isinstance(kilometers, int):
+        assert len(distance) == distance.km, (
+            f"'len()' for instance should return instance.km"
+        )
+    else:
+        assert kilometers == distance.km, (
+            f"'len()' for instance of float type should return instance.km"
```
Comment (danylott) [inline tests/test_main.py:426]: I believe this comment is not really correct.
`len` for instance of float type will be an error. Smth like `len(2.3)` -> error.
I believe the good message is `"'len()' for instance of Distance should return instance.km"`

File: tests/test_main.py [hunk 3]
```diff
@@ -363,11 +410,18 @@ def test_distance_class_le_number(kilometers, kilometers2, result):
...
     distance = Distance(kilometers)
-    assert len(distance) == distance.km, (
-        f"'len()' for instance should return instance.km"
-    )
+    if isinstance(kilometers, int):
+        assert len(distance) == distance.km, (
+            f"'len()' for instance should return instance.km"
+        )
+    else:
+        assert kilometers == distance.km, (
```
Comment (danylott) [inline tests/test_main.py:425]: In this test we check only `len` functionality. No this step needed

**Review by danylott — CHANGES_REQUESTED** [review]

File: tests/test_main.py [hunk 4]
```diff
@@ -363,11 +410,18 @@ def test_distance_class_le_number(kilometers, kilometers2, result):
...
+            f"'len()' for instance of float type should return instance.km"
+        )
```
Comment (Abnormaltype) [inline tests/test_main.py:427]: assert len(distance) == distance.km, (
        f"'len()' for instance should return instance.km"
    )

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: tests/test_main.py [hunk 5]
```diff
@@ -363,11 +410,18 @@ def test_distance_class_le_number(kilometers, kilometers2, result):
...
 def test_distance_class_len(kilometers):
     distance = Distance(kilometers)
-    assert len(distance) == distance.km, (
-        f"'len()' for instance should return instance.km"
-    )
+    if isinstance(kilometers, int):
+        assert len(distance) == distance.km, (
+            f"'len()' for instance should return instance.km"
+        )
+    else:
```
[the remark this answers is not in the document]
Comment (Nattalli) [reply tests/test_main.py:424]: > It seems that you don't check `len` method for float distance now

But method 'len' in python does not work for float type. It is return such error: 
"TypeError: object of type 'float' has no len()"

File: tests/test_main.py (hunk 1 above)
Comment (Abnormaltype) [reply tests/test_main.py:59]: this one

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Good job!

File: tests/test_main.py (hunk 1 above)
Comment (Abnormaltype) [inline tests/test_main.py:59]:
```suggestion
        (0.6, "Distance(km=0.6)")
```

**Review by Abnormaltype — CHANGES_REQUESTED** [review]
🥲

**Pull request merged**